### PR TITLE
feature/EN-68

### PR DESCRIPTION
**Pull request by yard**

### CODE REVIEW

File: build/goblin.js [hunk 1]
```diff
@@ -3764,13 +3771,13 @@ Goblin.HingeConstraint.prototype.update = (function(){
 
 		// 3,4 are rotational, constraining motion orthogonal to axis
 		world_axis.findOrthogonal( t1, t2 );
-		this.rows[3].jacobian[3] = -t1.x;
-		this.rows[3].jacobian[4] = -t1.y;
-		this.rows[3].jacobian[5] = -t1.z;
+		// this.rows[3].jacobian[3] = -t1.x;
```
Comment (stepan-beresnev) [inline build/goblin.js]: Can't see these lines in source files. Is it something you didn't want to commit?

File: src/classes/Shapes/BoxShape.js [hunk 2]
```diff
@@ -79,24 +79,9 @@ Goblin.BoxShape.prototype.findSupportPoint = function( direction, support_point
...
-		support_point.y = -this.half_height;
-	} else {
-		support_point.y = this.half_height;
-	}
-
-	if ( direction.z < 0 ) {
-		support_point.z = -this.half_depth;
-	} else {
-		support_point.z = this.half_depth;
-	}
```
Comment (stepan-beresnev) [inline src/classes/Shapes/BoxShape.js:99]: This might have been an optimization as calling sign function should be slower

**Review by stepan-beresnev — APPROVED** [review]

File: src/classes/Shapes/BoxShape.js (hunk 2 above)
Comment (yard) [reply src/classes/Shapes/BoxShape.js:99]: This is going to be roughly the same in terms of performance, however, support point will be closer to actual contact point, which helps GJK converge faster.

File: build/goblin.js (hunk 1 above)
Comment (yard) [reply build/goblin.js]: It's the built file diff, has been changed back and forth (source is at HingeConstraint.js)

**Pull request merged**